controllers/api.controller.js

```diff
@@ -81,3 +82,25 @@ exports.postCommentByArticle = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.patchVoteByArticle = async (req, res, next) => {
+  try {
+    const { article_id: articleId } = req.params;
+    if (isNaN(articleId)) {
+      return res.status(400).send({ msg: "Invalid article_id" });
+    }
+    if (!(await fetchArticleById(articleId))) {
```

Comment:
I wonder if the rejection logic should be in the `fetchArticleById` model. Is there a reason for this _not_ to reject?